docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tools for porting to .NET Core
+description: Learn about porting C++/CLI projects to .NET Core
+author: mjrousos
+ms.author: 
```

Comment:
```suggestion
ms.author: mikerou
```

Reply:
I actually left this one blank since I wasn't sure if I should be the ms.author or if it should be a dedicated UE person. I can work on docs intermittently in between customer engagements, but may not be able to respond promptly all the time. Is there a usual practice for when a 'non-UE' person submits docs?

Reply:
@mairaw What's your suggestion? We do need a value, otherwise we get a build warning.

Reply:
No need. Phillip is the ms.author for the porting folder
https://github.com/dotnet/docs/blob/master/docfx.json#L184